icaruscode/Timing/PMTTimingCorrectionsProvider.cxx

```diff
@@ -207,15 +201,13 @@ void icarusDB::PMTTimingCorrectionsProvider::readTimeCorrectionDatabase(const ar
     fDatabaseTimingCorrections.clear();
 
     ReadPMTCablesCorrections(run.id().run());
-
     ReadLaserCorrections(run.id().run());
-
-    //ReadCosmicsCorrections(run.id().run());
+    ReadCosmicsCorrections(run.id().run());
 
     if( fVerbose ) {
 
         mf::LogInfo(fLogCategory) << "Dump information from database " << std::endl;
-        mf::LogInfo(fLogCategory) << "channel, trigger cable delay, reset cable delay, laser corrections, muons correctsions" << std::endl;
+        mf::LogInfo(fLogCategory) << "channel, trigger cable delay, reset cable delay, laser corrections, muons corrections" << std::endl;
```

Comment:
You may consider here to merge the two messages, 
```cpp
      mf::LogInfo(fLogCategory) << "Dump information from database"
         << "\nchannel, trigger cable delay, reset cable delay, laser corrections, muon corrections";
```
and possibly to treat the message as described before for the one similar to this.

Reply:
I will do the same as in #626